check

```diff
@@ -265,7 +287,7 @@ def do_checks(srcdir):
         if 'domains' in data:
             errcnt += check_rdns(data)
             for domain in data.get('domains', []):
-                errcnt += check_dupe("Domain", domain, domains, community)
+                errcnt += check_dupe("Domain", domain.upper(), domains, community)
```

Comment:
We should either check for umlauts in the community file or (idna-)encode the domains here:
```
domain.upper().encode("idna")
```
Like @RubenKelevra suggested.